src/utils/url.ts

```diff
@@ -16,3 +16,12 @@ export const isSameURL = (url1: string, url2: string): boolean => {
     return false
   }
 }
+
+export const getHostnameFromTopLevelDomain = (url: string): string => {
+  let hostname
+  hostname = url.indexOf('://') > -1 ? url.split('/')[2] : url.split('/')[0]
+  hostname = hostname.split(':')[0]
+  hostname = hostname.split('?')[0]
+  hostname = hostname.split('.')
+  return `${hostname[hostname.length - 2]}.${hostname[hostname.length - 1]}`
+}
```

Comment:
I would strongly recommend against adding this ad-hoc URL parsing.
Maybe we could add a new config value like `topAppDomain`? (Not network-dependent)